### ISSUE TITLE
Clarity on suspend and resume in temporal model

### ISSUE THREAD

**Issue by jonopoltrack** [adlnet/xAPI-SCORM-Profile#32]
I believe the intent is as follows, but the fine details are not in the spec.  Add to temporal model section.

new attempt started with normal exit
initialized -> optionally do other stuff -> terminated

resume previous attempt with normal exit
resumed -> optionally do other stuff -> terminated

new attempt started but suspend on exit
initialized -> optionally do other stuff -> suspended

resume previous attempt and suspend on exit
resumed -> optionally do other stuff -> suspended.

For clarity, note that the last case, "resume previous attempt and suspend on exit" is NOT as follows:
resumed -> initialized -> optionally do other stuff -> suspended -> terminated

In other words, either use init or resume on start...  and suspend or terminate on finish

**Comment by creighton** [comment]
https://github.com/creighton/xAPI-SCORM-Profile/commit/c47fd0350c31d051c6edf216f6b54424a7872ee3
